docs/infrastructure/deployment-targets/dynamic-infrastructure/index.md

```diff
@@ -50,11 +48,19 @@ Only a subset of account types and deployment targets support being created dyna
 Before you can create dynamic targets in an Environment, the environment needs to be configured to allow it. See [Enabling dynamic infrastructure](/docs/infrastructure/deployment-targets/dynamic-infrastructure/index.md#enable-dynamic-infrastructure) for more information.
 :::
 
+:::hint
+**Targets from Step Packages**
+Octopus Deploy has recently developed a new architecture for deployment steps and targets, known as step packages. To learn more about step packages, read the [step package documentation](https://github.com/octopusdeploy/step-api#overview).
+
+Targets defined by step packages use the new generic `New-OctopusTarget` function to create targets. 
+:::
+
 - [Azure Web App](/docs/infrastructure/deployment-targets/dynamic-infrastructure/azure-web-app-target.md)
 - [Azure Service Fabric](/docs/infrastructure/deployment-targets/dynamic-infrastructure/azure-service-fabric-target.md)
 - [Azure Cloud Service](/docs/infrastructure/deployment-targets/dynamic-infrastructure/azure-cloud-service-target.md)
 - [Kubernetes Cluster](/docs/infrastructure/deployment-targets/dynamic-infrastructure/kubernetes-target.md)
-- [Remove Target](/docs/infrastructure/deployment-targets/dynamic-infrastructure/remove-target.md)
+- [AWS ECS Cluster](/docs/infrastructure/deployment-targets/dynamic-infrastructure/new-octopustarget.md)
```

Comment:
```suggestion
- [New Target](/docs/infrastructure/deployment-targets/dynamic-infrastructure/new-octopustarget.md)
```

Reply:
I've deliberately left this as the ECS cluster here taking them to New Target so people will more easily find what they are looking for. I think once we don't have "two ways" of doing this we can consolidate and simplify the list, but for now, I'd like to keep the targets listed separately. All new targets will just link to the same new target doc.

Reply:
Ah sorry I thought it was a typo 😃. I might have committed the change. I can revert tomorrow if you don't get chance to

Reply:
All good I'm already in there, I'll fix it up

Reply:
Thanks!